src/templates/Footer/index.js

```diff
@@ -112,7 +117,8 @@ function Footer(props) {
       <Layout
         className={layoutClasses}
         smallCols={[ 12 ]}
-        mediumCols={[ 4 ]}
+        mediumCols={[ 3, 3, 3, 6, 3 ]}
```

Comment:
Consider rewriting this as `mediumCols={[3]}` and then on the 4th col override it `Col mediumCols={6}`